reconcile/gitlab_housekeeping.py

```diff
@@ -549,7 +549,8 @@ def run(dry_run, wait_for_pipeline):
                     limit,
                     rebase,
                     pipeline_timeout,
-                    insist=True,
+                    # TODO: set insist=True after gitlab issues cleared
+                    insist=False,
```

Comment:
if `insist` is `False` here, then lower priority mrs could more likely to be merged before high priority ones.